### ISSUE TITLE
Can't scroll body of the page to access items at the bottom of a long admin menu

### ISSUE THREAD

**Issue by cameronjonesweb** [WordPress/gutenberg#11185]
**Describe the bug**
The body of the page doesn't scroll past the height of the Gutenberg editor (and following meta boxes). The means that if you have a number of plugins installed that add admin menu items, any menu items that are below the fold are completely inaccessible while viewing the Gutenberg editor. Having content in the editor beyond the height of the fold makes no difference.

**To Reproduce**
1. Register a number of admin menu items, enough so that the admin menu is taller than the fold of the browser window
2. Create a new post using the Gutenberg editor
3. Scroll down as far as you can

**Expected behavior**
The page should continue scrolling past the bottom of the Gutenberg editor content

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version 69.0.3497.100

**Additional context**
- Gutenberg 4.1.1

**Comment by cameronjonesweb** [comment]
Possibly related to: #8790, #8068

**Comment by jasonbahl** [comment]
Possibly related #11127

**Comment by designsimply** [comment]
I tested with Chrome 70 on Windows 10 and couldn't replicate the problem. Let me also check to make sure I understand the issue. Is the problem that if you add several plugins that add admin menu items, you cannot scroll the left side menu when you go to Posts > Add New?

@cameronjonesweb any chance you could double check whether this is still happening for you with the latest plugin version? (4.5.1) I tried testing a few different ways with different browsers and couldn't get scrolling to stop working and if that's happening I'd really like to help figure it out!

**Comment by designsimply** [comment]
Re-tested with WordPress 5.0-beta5-43909 using Firefox 63.0.1 on macOS 10.13.6 and still can't replicate! Closed #12195 as a duplicate. @mervb can you spot anything I'm missing in my testing? See video: [1m17s](https://cloudup.com/c_AYgaipebv).

**Comment by cameronjonesweb** [comment]
My experience is that it's sporadic, sometimes I'm affected but I've also noticed sometimes it's fine. I have a feeling it may be related to the size of the window. I'll do some more testing for you to see if I can still reproduce the bug.

**Comment by spencerfinnell** [comment]
There are a lot of scrolling issues still in 4.5.1. Testing using Chrome Version 70.0.3538.102 (Official Build) (64-bit) macOS 10.14

- Sometimes the post loads cut off and it's impossible to scroll all the way up to see the content or post title. The same goes for the bottom. The editor window will scroll separately but never able to reach the bottom.
- Scrolling over the admin sidebar scrolls the main window only (none of Gutenberg)
- After having clicked the sidebar or outside of the editor, the space key scrolls the main window only (none of Gutenberg).

Very randomly do multiple sidebars appear, but it's very hard to know what is causing it when all the editor content isn't accessible so it's hard to even tell where I am supposed to be on the document.

**Comment by spencerfinnell** [comment]
Removing

```css
.edit-post-layout, .edit-post-layout__content {
    height: 100%;
}
```

Allows scrolling to the bottom of the editor, but makes scrolling the settings sidebar a little more difficult.

**Comment by spencerfinnell** [comment]
This should show the problem pretty clearly: https://cloudup.com/cvsKkaxN8he

When the cursor is over the editor, the editor scrolls. When the cursor is over the sidebar, the document scrolls. If those get out of sync then content becomes unreachable.

**Comment by designsimply** [comment]
Noting a past report from @MYIG showing the problem @spencerfinnell noted where "the post loads cut off and it's impossible to scroll":

![cap 2018-11-18 15-31-13-727](https://user-images.githubusercontent.com/44285360/48773795-c4fae400-ed0b-11e8-8594-16d1a337f1f7.png)
![cap 2018-11-18 15-31-55-000](https://user-images.githubusercontent.com/44285360/48773796-c4fae400-ed0b-11e8-893b-cd9ef6788396.png)

> Windows 10 1803, chrome 70.0.3538.102, wordpress 4.9.8 and gutenberg 4.

**Comment by mervb** [comment]
It is still impossible to scroll in (5.0-RC3-43966)

Very easy to reproduce the error as explained by several posts. I discovered this issue immediately after using WordPress 5.0 for 5 minutes. 

90% of users have a large number of menu items on the left and with the new editor the menu panel on the left has to be scrolled up and down in order to reach the bottom of the post to make edits or access additional panel information.

TO REPRODUCE THE ERROR

1. Copy a large amount of text broken into multiple paragraphs and paste into a new post. 
2. Ensure you have a large number of plugins active that make the left hand menu long. Ie 10 custom post types.
3. Publish (not necessary, error still present)
4. Edit post, attempt to scroll to the end.
5. Not possible....

**Comment by miscs** [comment]
same problem here... please fix this before release. I fear a lot of disappointed Gutenberg reviews because of this... It´s really a problem with real world content (-> long articles).

**Comment by jasonbahl** [comment]
I updated to WordPress 5.0 and this is _still_ an issue.

See screen recording below: 

![scroll-issue-wp-5 0](https://user-images.githubusercontent.com/1260765/49616441-7473bf80-f976-11e8-9fe9-0081cafc2ecb.gif)

You can see that the main content pane is not scrollable up to the top of my content. I have to move my mouse over to the menu and scroll up to be able to scroll up to the top of the content.

As a user, I expect to be able to scroll up and down on the blocks to get to the top and bottom of the contents of my blocks.

**Comment by chrisvanpatten** [comment]
@jasonbahl #12644 was opened today which may solve this, and seems like something that could land in 5.0.1. I plan to test hopefully tomorrow as I've also experienced this, but more eyes on the PR could help too :)

**Comment by youknowriad** [comment]
Is this solved by #12644?

**Comment by themightymo** [comment]
+1 I'm experiencing this issue as well (and have been for a long time).

**Comment by designsimply** [comment]
Everyone, if you test with the 4.8 plugin, is the problem resolved for you? See https://make.wordpress.org/test/2019/01/04/call-for-testing-gutenberg-4-8/ and if you leave a comment there noting if you've tested item 6 that would be helpful!

**Comment by christinablust** [comment]
After encountering this issue just now and finding this related ticket, I turned on the 4.8 plugin and am fairly certain the scroll behavior has been corrected. BUT activating Gutenberg 4.8 (with no other changes) made my Advanced Custom Fields metaboxes below the editor disappear entirely. Unsure if this is related to the solution to 11185, or a completely new problem.

**Comment by cameronjonesweb** [comment]
Unsure if it's directly related but https://wordpress.org/gutenberg/ now doesn't scroll at all

**Comment by designsimply** [comment]
@christinablust thanks for the note, I'll check on ACF and make a note in the comments on https://make.wordpress.org/test/2019/01/04/call-for-testing-gutenberg-4-8/

@cameronjonesweb I bet that's unrelated. Thank you for the note and I will look into it!

**Comment by youknowriad** [comment]
@christinablust Are you using the last ACF version?

**Comment by christinablust** [comment]
@youknowriad yes, I'm in ACF Pro 5.7.9.

**Comment by youknowriad** [comment]
Interesting, Please feel free to open a separate issue if there's a way to reproduce this easily (include all the details you can). When releasing I did test with ACF and the fields were appearing properly. (I tested with the free version though)

**Comment by youknowriad** [comment]
Closing as this appears solved by  #12644

**Comment by maximummax** [comment]
just can't scroll down the page when using gutenberg editor. WP 5.2.2, chrome & FF on osx. Please fix this is blocking!!!!!!!!!! Sucks.

**Comment by maximummax** [comment]
thank you automattic, i cannot fucking publish a post which content "goes out" of my screen. Are you serious?

**Comment by designsimply** [comment]
@maximummax please open a support request at https://wordpress.com/help/contact if you are using WordPress.com or at https://wordpress.org/support/forum/how-to-and-troubleshooting/ if you are using WordPress elsewhere. It sounds to me like you are using WordPress somewhere other than WordPress.com and you may have a plugin conflict. Please ask support to help you learn how to confirm whether the problem is related to a plugin—they should also be able to help file an bug report for you in the right place but some basic troubleshooting steps should be checked first. I'm sorry for the trouble you're having!

**Comment by maximummax** [comment]
Hi Sheri,

Thx for the answer. Will file a bug as advised!
This is indeed related to a self-hosted wp install.

Cheers,
Max

Le ven. 16 août 2019 à 23:46, Sheri Bigelow <notifications@github.com> a
écrit :

**Comment by maximummax** [comment]
i deactivated all plugins and the bug's still there ..... hmmmm

**Comment by maximummax** [comment]
ok, i caught the culprit, it's the theme. It's called Elision from Qode. https://themeforest.net/item/elision-retina-multipurpose-wordpress-theme/6382990
i 'm gonna head to those guys to file a bug since it apparently happens on their side.

**Comment by maximummax** [comment]
if someone around here wants to check this one, just message me, I'll give you access to the site admin of the website (i mirrored it on a dev server to check the issue, so feel free to play around if you like)

**Comment by maximummax** [comment]
will keep you posted anyway

**Comment by maximummax** [comment]
ok, so here's the end of the story. the bug's been fixed with the 4.3 release of the theme.
![Screen Shot 2019-08-27 at 09 18 37](https://user-images.githubusercontent.com/6917173/63750301-7f2c0700-c8ad-11e9-84b7-6969eb881498.png)
thanks everybody, sorry for the spam.
